repository/sql.go

```diff
@@ -175,11 +176,11 @@ func (sql *sqlRepository) DeleteRelation(id string) error {
 	return nil
 }
 
-// FindAssetByContent finds assets in the database that match the provided asset data.
+// FindAssetByContent finds assets in the database that match the provided asset data and last seen after the since parameter.
```

Comment:
I think we should add a line in here that describes the behavior when Zero time is passed. something like:

`If since.IsZero(), the parameter will be ignored`

Reply:
and also add this to the other methods.

Reply:
Will do